### PR TITLE
chore: update anthos-edge-usecases repo name

### PR DESCRIPTION
**Pull request by Shabirmean**
Fixes None
- Anthos/GKE DPE team is renaming the [anthos-edge-usecases](https://github.com/GoogleCloudPlatform/anthos-edge-usecases) repo to `point-of-sale`
- Thus we would like the sloth sheets updated so that when the change happens we can merge it

### CODE REVIEW

**Review by bcoe — APPROVED** [review]

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
we will need to add a `sync-repo-settings` override, since this repo requires `>=Node 12`.

**Comment by Shabirmean** [comment]
> we will need to add a `sync-repo-settings` override, since this repo requires `>=Node 12`.

Hey @bcoe - does this mean I need to update some config? Or is this PR good to be merged?

**Comment by tmatsuo** [comment]
@Shabirmean I added that file. I'm going to merge this.

**Comment by Shabirmean** [comment]
> @Shabirmean I added that file. I'm going to merge this.

Thank you!!

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [12, 14, 16]
+        node: [10, 12, 14, 16]
```
Comment (tmatsuo) [inline .github/workflows/ci.yaml:12]: Why are you adding node 10 here?

Comment (tmatsuo) [reply .github/workflows/ci.yaml:12]: I'm going to remove it

Comment (Shabirmean) [reply .github/workflows/ci.yaml]: It was added by Ben I think

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**